### ISSUE TITLE
add startOf / endOf methods

### ISSUE THREAD

**Issue by d1rtym0nk3y** [CounterMarch/moment.cfc#8]
Per the moment.js docs
http://momentjs.com/docs/#/manipulating/start-of/

These are really useful mutators when working with a moment

**Comment by atuttle** [comment]
Yeah, these are something I intended to get to, but I can't yet. Pull requests welcome. Please model your tests after the current tests for utc(). :)

**Comment by artknight** [comment]
+1
